### PR TITLE
Android: Add Spanish Translation

### PR DESCRIPTION
**Pull request by Keropon**

### CODE REVIEW

File: build_android/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<!-- Application Navigation -->
+
+	<string name="about_play">Acerca de Play!</string>
+	<string name="about_play_emu">Play! Emulador de PlayStation 2</string>
```
Comment (Croden1999) [inline build_android/src/main/res/values-es/strings.xml:7]: here is should `Play! PlayStation 2 Emulator`

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: build_android/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (Keropon) [reply build_android/src/main/res/values-es/strings.xml:7]: Ok, done. So this string shouldn't be translated... 🤔

**Comment by jpd002** [comment]
Thanks a bunch for this!

**Comment by Keropon** [comment]
You're welcome, you don't know how happy the 256 emulation made me. It's the least I could do.

**Pull request merged**